### ISSUE TITLE
Genesis needs to tar up the bin

### ISSUE THREAD

**Issue by anwayde** [firedancer-io/firedancer#1600]
When a validator joins and looks to fetch the genesis, it looks for `genesis.tar.bz2`, so our genesis stage needs to generate that.

**Comment by ripatel-fd** [comment]
This task is pretty much impossible to do in the C part of our codebase unless we do it in a hacky way.
The genesis archive doesn't only contain the genesis bin but also a RocksDB archive containing the shreds of the genesis block. Other validators can't boot using a genesis.tar.bz2 that only contains the genesis.bin file.
I suggest closing this task and letting other validators boot of a snapshot instead.

cc @mmcgee-jump

**Comment by mmcgee-jump** [comment]
No, we can do this as part of the blockstore.c init step, which runs in 1.5 and does produce this rocksdb data.

**Comment by ripatel-fd** [comment]
deps.sh doesn't install bzip2 though.
